Good-Bids/goodbids
# [#762] Delete Accounts

# Summary

This PR adds the Delete Me plugin and ability for Customers to Delete their account.

## Issues

* #762 

## Testing Instructions

1. As a Customer user, visit My Account > Account Details
2. Input your user password and click Confirm Deletion
3. Your account is now deleted.

## Screenshots

<img width="849" alt="Screenshot 2024-03-18 at 10 29 23 AM" src="https://github.com/Good-Bids/goodbids/assets/122309362/faa52953-5685-482f-8001-d0e4b48dd3cf">
